File: ValidAnagram/ValidAnagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
        	return False
        n = len(s)
        counts = [0] * 26
        for i in xrange(n):
        	counts[ord(s[i] - 97)] += 1
        	counts[ord(s[i] - 97)] -= 1
        for i in counts:
        	if i != 0:
        		return False
        return True

File: ValidAnagram/ValidAnagram.cpp
#include<string>
using namespace std;

class Solution {
public:
	bool isAnagram(string s, string t) {
		if (s.size() != t.size())
			return false;
		int counts[26] = { 0 };
		for (int i = 0; i < s.size(); i++){
			counts[s[i] - 'a'] += 1;
			counts[t[i] - 'a'] -= 1;
		}
		for (int i = 0; i < 26; i++){
			if (counts[i] != 0)
				return false;
		}
		return true;
	}
};